php56/root/etc/confd/templates/xdebug.ini.tmpl

```diff
@@ -8,12 +8,15 @@ xdebug.coverage_enable=0
 xdebug.default_enable=1
 
 xdebug.remote_enable=1
+xdebug.remote_autostart=1
 xdebug.remote_connect_back=1
-xdebug.remote_host=localhost
+; If the following remote_host value does not work use the environmental
+; variable XDEBUG_CONFIG to override it. Documentation for format at
+; https://xdebug.org/docs/remote
+xdebug.remote_host=192.168.99.1
```

Comment:
Since changing this seems like it would be common, could we have a specific example of that override in the README? (We should also get the Outrigger Xdebug doc revised to include it later.)